src/extract.jl

```diff
@@ -296,8 +296,8 @@ function git_file_hash(
     # where you write data to an IO object and it maintains a hash
     padded_size = round_up(size)
     while padded_size > 0
-        max_read_len = Int(min(padded_size, length(buf)))
-        read_len = readbytes!(tar, buf, max_read_len)
+        max_read_len = Int(min(padded_size, length(buf)))::Int
+        read_len = Int(readbytes!(tar, buf, max_read_len))::Int
```

Comment:
Do you actually need the annotations of `::Int` now?  I would think that actually doing `Int()` would be enough.

Reply:
I thought so, too, and just used `Int(...)`. However, I still got the invalidations with that :confused:

Reply:
https://github.com/JuliaLang/julia/issues/42372

Reply:
@staticfloat Looks like we can't do anything better than this right now.